fetch/api/resources/trickle.py

```diff
@@ -9,6 +11,6 @@ def main(request, response):
     response.headers.set("Content-type", "text/plain")
     response.write_status_headers()
     time.sleep(delay)
-    for i in xrange(count):
+    for i in range(count):
```

Comment:
Sure. The only thought here is that - would it make sense to use python 3 API as much as possible?

Reply:
I think it's better to prioritise not changing the behaviour rather than picking the things that are closest in name to Python 3. If we ever drop Py2 support then we're going to need to do a pass removing all the `six` stuff and at that point converting `xrange` into `range` will be obvious.

Reply:
Right. Updated the Code. Thank you!